user-service/src/main/java/com/spike/user/controller/UserController.java

```diff
@@ -407,4 +407,22 @@ public ResponseEntity<Object> getAllContacts(){
     	return ResponseHandler.responseBuilder("show all contacts ", HttpStatus.OK, contactsDtoList);
     }
 
+    @DeleteMapping("/delete-contact/{contactId}")
+    public ResponseEntity<Object> deleteContact(@PathVariable Long contactId){
+	   logger.info("Received request to delete contact with ID: {}", contactId);
```

Comment:
don't we have exceptional handler for handling all these exceptions at one place